src/graph/YieldExecutor.cpp

```diff
@@ -446,8 +446,9 @@ void YieldExecutor::feedResult(std::unique_ptr<InterimResult> result) {
 }
 
 void YieldExecutor::setupResponse(cpp2::ExecutionResponse &resp) {
-    CHECK(resp_ != nullptr);
-    resp = std::move(*resp_);
+    if (resp_ != nullptr) {
```

Comment:
Here maybe an empty resp should be set to return to the caller, similar to the GoExecutor :: setupResponse method？

Reply:
`GoExecutor::setupResponse ` create an empty object is redundant, wdyt?